src/async_wrap.cc

```diff
@@ -63,7 +63,7 @@ namespace node {
 static const char* const provider_names[] = {
 #define V(PROVIDER)                                                           \
   #PROVIDER,
-  NODE_ASYNC_PROVIDER_TYPES(V)
+    NODE_ASYNC_PROVIDER_TYPES(V)
```

Comment:
Can you undo the unrelated whitespace changes?

Reply:
sure, seems visual studio does some autoformat and I missed this one.